spec/operators/bufferTime-spec.js

```diff
@@ -59,7 +59,7 @@ describe('Observable.prototype.bufferTime', function () {
     var values = {
       w: ['a','b']
     };
-    var e1 =   hot('---a---b---c---#---e---f---g---|');
+    var e1 =   hot('---a---b---c---#');
```

Comment:
We shouldn't have written that kind of hot observable marble diagram in the first place.